doc/parameters.md

```diff
@@ -34,6 +35,8 @@ Lazy will load it on demand for each incoming request (to guarantee a complete r
 
 ### `APICAST_CUSTOM_CONFIG`
 
+**Deprecated:** Use [policies](./policies.md) instead.
```

Comment:
Maybe we should put a link to https://github.com/3scale/apicast-example-policy somewhere? Maybe in the `policies.md` itself...

Reply:
If, then in the policies.md. Definitely not here.

Reply:
@mayorova maybe https://github.com/3scale/apicast/pull/723 would good place to link that.